### DISCUSSION TITLE
Error: pandoc document conversion failed with error 22

### DISCUSSION THREAD

**Discussion by robertprince** [jgm/pandoc#10357] [Q&A]
I am trying to save a docx file and am getting the following message: 

Unknown output format doc
Pandoc can convert to DOCX, but not to DOC.
Error: pandoc document conversion failed with error 22.

My code is:

library(car)
library(tidyverse)
library(gtsummary)
library(flextable)
library(officer)
library(gt)

tab_1 <-
  gtcars |>
  dplyr::select(model, year, hp, trq) |>
  dplyr::slice(1:5) |>
  gt(rowname_col = "model") |>
  tab_stubhead(label = "car")

tab_1

tab_1 |> gtsave("tab_1.docx")

**Comment by tarleb** [comment]
This looks like an issue with officr or RMarkdown. You'll probably get better answers if you asks people from those projects. The Posit forum could be an appropriate choice: https://forum.posit.co
